### DISCUSSION TITLE
Problem in Documents Numbers

### DISCUSSION THREAD

**Discussion by MasoudDorvash** [jimmejardine/qiqqa-open-source#430] [Q&A]
Hi everyone
I would like to ask about a problem I'm experiencing with Qiqqa regarding the total number of documents. As shown in 
![126 out](https://github.com/jimmejardine/qiqqa-open-source/assets/164001442/1ac4a9c1-744d-4c70-989f-fae0d5a2d1fe)
the total number of documents in My Library is indicated as 126. However, when I enter, it shows 122 documents, as seen in 
![122 in](https://github.com/jimmejardine/qiqqa-open-source/assets/164001442/4fcd328f-9e75-45b9-8017-bbac9461a7d0)
The missing 4 documents are those that I've removed from the library, directory, and watch folder.
![122 directory](https://github.com/jimmejardine/qiqqa-open-source/assets/164001442/583e3715-0fc6-4f7e-98f7-b4dfc378d50a)
Yet, it still indicates that there are 126 documents. How can I resolve this issue?

**Comment by zhangxiaoxing** [comment]
There is a "Purge deleted PDFs" button in the config menu, maybe that would help.
![image](https://github.com/jimmejardine/qiqqa-open-source/assets/58493968/400974c3-39a5-4dfa-8103-44c09b976e0c)

**Reply by MasoudDorvash** [reply]
Thanks for your help, but that doesn't work either
![Untitled](https://github.com/jimmejardine/qiqqa-open-source/assets/164001442/dc09d8a5-72fa-4eaf-9e4b-08d6e9bae377)
